Dockerfile

```diff
@@ -25,8 +25,8 @@ RUN buildDeps=' \
     \
     && curl -fsSL ${url}.asc -o ${tmpdir}/rakudo.tar.gz.asc \
     && curl -fsSL $url -o ${tmpdir}/rakudo.tar.gz \
-    && gpg --keyserver $keyserver --recv-keys $keyfp \
-    && gpg --batch --verify ${tmpdir}/rakudo.tar.gz.asc ${tmpdir}/rakudo.tar.gz \
+    && gpg --no-tty --keyserver $keyserver --recv-keys $keyfp \
+    && gpg --no-tty --batch --verify ${tmpdir}/rakudo.tar.gz.asc ${tmpdir}/rakudo.tar.gz \
```

Comment:
This should be just `--batch` on both lines (no need for `--no-tty` with `--batch`).

See https://bugs.debian.org/913614 and all the linked things on https://github.com/docker-library/busybox/pull/55 for more information.

Reply:
@tianon could we also just inline the GPG pubkey here (and avoid using keyserver for this altogether?)

Reply:
Changed, and we still have the error.